### ISSUE TITLE
Script error 

### ISSUE THREAD

**Issue by gtumanyan** [ionuttbara/edge-remover#22]
In UninstallOnedrive.bat a quote is missing at the end of line:165

**Comment by maulana-kurniawan** [comment]
Yep, running `Edge.Remover.exe` resulted in error.

Screenshot attached:

![Screenshot 2023-08-11 050704](https://github.com/ionuttbara/edge-remover/assets/117083636/e2ebbb45-141f-4b97-b8ea-8a936b98d867)

**Comment by ionuttbara** [comment]
ahh.. it removes the edge, but the message its not shown.. so a fix is coming

**Comment by seba2282** [comment]
To me showed that communicat but Edge I saw still like didn't worked. So fix is needed. :)

**Comment by ionuttbara** [comment]
i need to repackage , but i am modify @AveYo 's implementation first... next week i will release a version with provided fix!

**Comment by seba2282** [comment]
Thanks for informations and sorry if i made some trouble by my ask. :)

**Comment by ionuttbara** [comment]
no problem.. 
the trouble is my inactivity at github, i take a break from coding and blah blah =)

**Comment by seba2282** [comment]
I tried too that shadow whisperer moment and that remove edge, WebView, but not that edge update. I tried "only edge" and that gui with uncheck WebView, so shouldn't delete WebView. So not only you have problems with scripts. If i can tell true. But after what Microsoft will add to version 117 (make screenshots every sites), can tell bye to edge. Firefox is good to replace but few sad haven't translator integrated how that all chromium. If about mine opinion, you make better scripts to remove edge.
